pages/spacedeck/[[...roomId]].js

```diff
@@ -140,21 +142,51 @@ export default function Spacedeck() {
         };
     }, [spacedeck]);
 
-    async function createSketchRoom(link, name, parent = serviceSpaceId) {
-        // eslint-disable-next-line no-undef
-        logger.debug('creating room for ' + name);
-        const room = await matrix.createRoom(name, false, '', 'invite', 'content', 'spacedeck').catch(() => {
-            setErrorMessage(t('Something went wrong when trying to create a new room'));
-        });
-        await auth.getAuthenticationProvider('matrix').addSpaceChild(parent, room).catch(() => {
-            setErrorMessage(t('Couldn\'t add the new room to your sketch folder'));
-        });
-        await matrixClient.sendMessage(room, {
-            msgtype: 'm.text',
-            body: link,
-        }).catch(() => {
-            setErrorMessage(t('Something went wrong when trying to save the new sketch link'));
-        });
+    async function createSketchRoom(link, name, parent = serviceSpaceId, retries = 0) {
+        // Log debugging information
+        setUserFeedback(t('Syncing {{name}} from server', { name: name }));
+
+        // Create the room with retry handling
+        const createRoomForSketch = async (retries = 1) => {
```

Comment:
Is the `retries` parameter ever being used? 🤔

Reply:
it is being used for debug logs which is quite useful.

Reply:
But it's always "1", no?

Reply:
🤦 missed that when refactoring the code. removed it for now.